dts/bindings/gpio/ite,it8xxx2-gpio-v2.yaml

```diff
@@ -27,6 +27,12 @@ properties:
   wuc-mask:
     type: array
 
+  extended-set:
+    type: array
+    description: |
+      gpioksi, gpioksoh and gpioksol require extended setting
+      when used as GPIO output.
```

Comment:
Perhaps add a reference to where to find these settings, which register they are written to etc.?

Reply:
Description added.